docs/process/metrics.md

```diff
@@ -0,0 +1,51 @@
+| `usage` | Constant used to group/filterin data warehouse | Specifies that this data is related to GitHub Desktop usage, so we can filter it correctly in our analytics. |
+
+## Measures
+These are general metrics about general feature usage and specific feature behaviors. These help us understand our users' mental map of the application, hypothesize pain points within the application, and aid us in feature and bug fix planning so that we can improve workflows that might actually be used.
+
+| Metric | Description | Justification |
+|:--|:--|:--|
+| `commits` | The number of commits made. | To understand usage patterns of commits made in Desktop. |
+| `openShellCount` | The number of times the user has opened a shell from the app. | To understand if people need to use the command line because of missing features. |
+| `partialCommits` | The number of partial commits. | To understand usage patterns of commits made in Desktop. |
+| `coAuthoredCommits` | The number of commits created with one or more co-authors. | To understand usage patterns of commits made in Desktop. |
+| `branchComparisons` | The number of times a branch is compared to an arbitrary branch. | To understand usage patterns around the compare branches feature. |
+| `defaultBranchComparisons` | The number of times a branch is compared to `master`. | To understand usage patterns around the compare branches feature. |
+| `mergesInitiatedFromComparison` | The number of times a merge is initiated in the `compare` sidebar. | To understand usage patterns around the compare branches feature. |
+| `updateFromDefaultBranchMenuCount` | The number of times the `Branch -> Update From Default Branch` menu item is used. | To understand usage patterns around the compare branches feature. |
+| `mergeIntoCurrentBranchMenuCount` | The number of times the `Branch -> Merge Into Current Branch` menu item is used. | To understand usage patterns around the compare branches feature. |
+| `prBranchCheckouts` | The number of times the user checks out a branch using the PR menu. | To understand usage patterns around the PR checkout menu. |
+| `repoWithIndicatorClicked` | The numbers of times a repo with indicators is clicked on repo list view. | To understand usage patterns around the additional information in the repository list feature. |
+| `repoWithoutIndicatorClicked` | The numbers of times a repo without indicators is clicked on repo list view.  | To understand usage patterns around the additional information in the repository list feature. |
+| `divergingBranchBannerDismissal` | The number of times the user dismisses the diverged branch notification. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `divergingBranchBannerInitatedMerge` | The number of times the user merges from the diverged branch notification merge CTA button. | To understand usage patterns around the notification of diverging from the default branch feature. |
```

Comment:
Ugh, just spotted the missing `i` in `Initated` in `divergingBranchBannerInitatedMerge` but that's also in the source so that ship has sailed...

Reply:
We should be able to easily rename it once we move away from Dexie.

Reply:
I will keep it as is to match the source